packages/block-library/src/query-pagination-next/block.json

```diff
@@ -9,6 +9,10 @@
 	"attributes": {
 		"label": {
 			"type": "string"
+		},
+		"arrow": {
```

Comment:
If I'm understanding the PR properly, the arrow is meant to be saved in the "query" context right? If that's the case, why do we need an attribute here? seems redundant?

Reply:
I followed the proposed approach in your [below comment](https://github.com/WordPress/gutenberg/pull/33656#discussion_r699991279).